src/mediator/mediator.c

```diff
@@ -2898,13 +2907,13 @@ static void *start_pcap_thread(void *params) {
     if (pstate.packet) {
         trace_destroy_packet(pstate.packet);
     }
-    logger(LOG_INFO, "OpenLI mediator: exiting pcap thread.");
+    logger(LOG_INFO, "OpenLI Mediator: exiting pcap thread.");
     pthread_exit(NULL);
 }
 
 int main(int argc, char *argv[]) {
     char *configfile = NULL;
-    char *mediatorid = NULL;
+    int *mediatorid = NULL;
```

Comment:
This should be an "int" not an "int *".

Reply:
This also means that you'll need to use `strtoul` when you assign it inside the getopt block, e.g.

    case 'm':
        mediatorid = strtoul(optarg, NULL, 0);
        break;